charts/cryostat/templates/tests/test-grafana-connection.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "cryostat.labels" . | nindent 4 }}
     app.kubernetes.io/component: test-grafana-connection
+    helm-test: cryostat
```

Comment:
Just curious, are there some reasons to add this label?

Reply:
Just to make cleanup of the test Pods easier.

Reply:
Sure, how about using having standard format for cryostat-specific label? Like below or similar options:

```
charts.cryostat.io/role: helm-test
```